src/components/Carousel.tsx

```diff
@@ -1,18 +1,176 @@
-import React from 'react';
+import cn from 'classnames';
+import React, { useState } from 'react';
 import './Carousel.scss';
 
-const Carousel: React.FC = () => (
-  <div className="Carousel">
-    <ul className="Carousel__list">
-      <li><img src="./img/1.png" alt="1" /></li>
-      <li><img src="./img/1.png" alt="2" /></li>
-      <li><img src="./img/1.png" alt="3" /></li>
-      <li><img src="./img/1.png" alt="4" /></li>
-    </ul>
-
-    <button type="button">Prev</button>
-    <button type="button">Next</button>
-  </div>
-);
+type Props = {
+  images: string[],
+};
+
+const Carousel: React.FC<Props> = ({ images }) => {
+  const [currentIndex, setCurrentIndex] = useState(0);
```

Comment:
Add a generic types everywhere for state 
```suggestion
  const [currentIndex, setCurrentIndex] = useState<number>(0);
```